src/docs/null-safety.md

```diff
@@ -54,6 +56,22 @@ check out these pages:
   in which only some libraries are null safe.
 * [FAQ][]:
   Questions that have come up during migration to null safety.
+  
+## Known issues
+
+Not all parts of the Flutter SDK support null safety yet,
+as some parts still need additional work to
+[migrate to null safety]({{site.dart-site}}/null-safety/migration-guide).
+
+We're currently aware of the following issues:
```

Comment:
Should we specify that even depending on `flutter_driver` may limit you from picking up other dependencies that have migrated?

Reply:
Thanks, took your change below